diff -urN orig/build_defs/internal_do_not_use/j2cl_js_common.bzl new/build_defs/internal_do_not_use/j2cl_js_common.bzl
--- orig/build_defs/internal_do_not_use/j2cl_js_common.bzl	2026-07-15 10:00:00
+++ new/build_defs/internal_do_not_use/j2cl_js_common.bzl	2026-07-30 10:00:00
@@ -90,6 +90,7 @@
         data,
         test_class,
         tags,
+        compilation_level = "DEFAULT",
         default_browser = None,
         browsers = [],
         **args):  # @unused
@@ -142,11 +143,14 @@
     if not browsers:
         compile = True

+    if compilation_level == "DEFAULT":
+        compilation_level = "ADVANCED" if compile else "BUNDLE"
+
     closure_js_test(
         name = name,
         srcs = [":%s" % testsuite_file_name],
         deps = deps,
-        compilation_level = "ADVANCED" if compile else "BUNDLE",
+        compilation_level = compilation_level,
         browsers = browsers,
         data = data,
         testonly = 1,
diff -urN orig/build_defs/internal_do_not_use/j2cl_test_common.bzl new/build_defs/internal_do_not_use/j2cl_test_common.bzl
--- orig/build_defs/internal_do_not_use/j2cl_test_common.bzl	2026-07-15 10:00:00
+++ new/build_defs/internal_do_not_use/j2cl_test_common.bzl	2026-07-30 10:00:00
@@ -75,5 +75,6 @@
     "browser_overrides",
     "compiler",
+    "compilation_level",
     "default_browser",
     "deprecation",
     "deps_mgmt",
@@ -97,5 +98,6 @@
     "browser_overrides",
     "compiler",
+    "compilation_level",
     "default_browser",
     "deps_mgmt",
     "distribs",
