diff --git a/build_defs/internal_do_not_use/j2cl_java_library.bzl b/build_defs/internal_do_not_use/j2cl_java_library.bzl
--- a/build_defs/internal_do_not_use/j2cl_java_library.bzl
+++ b/build_defs/internal_do_not_use/j2cl_java_library.bzl
@@ -119,3 +119,6 @@
-    extra_providers = [DefaultInfo(files = depset(outputs))]
+    extra_providers = [
+        DefaultInfo(files = depset(outputs)),
+        j2cl_provider._private_.java_info,
+    ]
     if j2kt_provider:
         extra_providers.append(j2kt_provider)
