diff --git a/core/src/main/java/zemeckis/BUILD.bazel b/core/src/main/java/zemeckis/BUILD.bazel
index ac5edc1..dc78a47 100644
--- a/core/src/main/java/zemeckis/BUILD.bazel
+++ b/core/src/main/java/zemeckis/BUILD.bazel
@@ -118,10 +118,10 @@ j2cl_library(
     ],
     experimental_enable_jspecify_support_do_not_enable_without_jspecify_static_checking_or_you_might_cause_an_outage = True,
     deps = [
-        "//third_party/java:braincheck_core-j2cl",
-        "//third_party/java:grim_annotations-j2cl",
-        "//third_party/java:jsinterop_annotations-j2cl",
-        "//third_party/java:jspecify-j2cl",
-        "//third_party/java:org_jetbrains_annotations-j2cl",
+        "@//third_party/java:braincheck_core-j2cl",
+        "@//third_party/java:grim_annotations-j2cl",
+        "@//third_party/java:jsinterop_annotations-j2cl",
+        "@//third_party/java:jspecify-j2cl",
+        "@//third_party/java:org_jetbrains_annotations-j2cl",
     ],
 )
